whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -545,11 +543,11 @@ private RpslObject getRpslObjectByAttributeType(final CIString attributeValue, f
     }
 
     private Entity createEntity(final RpslObject rpslObject, final String requestUrl) {
-        // top-level entity has no role
-        return createEntity(rpslObject, null, requestUrl);
+        // top-level entity has no role and has unfiltered information
+        return createEntity(rpslObject, null, requestUrl, false);
```

Comment:
Just an option I think much cleaner, add the email entity directly on top level instead of passing filtered boolean flag.

Reply:
I passing this for the contact entities. We need to avoid adding the email in the contact entities.
And this method is used for both, for creating the vcard of the top level entity and to create the vcard for the contact entities.
I think that it is better to remain having one place to create the vcard, and let that method decide if that vcard need the email or not